webdriver/client.py

```diff
@@ -349,10 +350,14 @@ def send_raw_command(self, method, url, body=None, headers=None):
         :return: an instance of wdclient.Response describing the HTTP response
             received from the remote end
         """
-        url = urlparse.urljoin("session/%s/" % self.session_id, url)
-        return self.transport.send(method, url, body, headers)
+        response = self.transport.send(method, url, body, headers)
+        if response.status != 200:
+            value = response.body["value"]
+            cls = error.get(value.get("error"))
+            raise cls(value.get("message"))
```

Comment:
@jgraham pushed fixups to your original review that I felt addressed most of the issues. I submitted https://github.com/w3c/wdclient/pull/10, pending your review, to fix the one outstanding issue regarding a falsy value of `response.body["value"]`.

I agree moving the exception raising to `send_raw_command` was a mistake as this will break some of the existing tests. I filed https://github.com/w3c/wdclient/pull/11, superseding the PR I mentioned above, to address this. Apologies for not spotting this earlier.